advanced_filters/static/advanced-filters/advanced-filters.js

```diff
@@ -139,7 +139,7 @@ var OperatorHandlers = function($) {
 			$(this).off("change");
 		});
 		$('.form-row input.query-value').each(function() {
-			$(this).select2("destroy");
+			// $(this).select2("destroy");
```

Comment:
I'd keep this change out of this PR, we haven't got e2e tests implemented yet, so I'm not sure what behaviour this introduces. A separate PR with some more details can be useful.

Reply:
Yes, The issue is I am using most recent select2 JS file:

```Python
SELECT2_JS = '//cdnjs.cloudflare.com/ajax/libs/select2/4.0.13/js/select2.min.js'
```

https://pythonhosted.org/Django-Select2/django_select2.html#django_select2.conf.Select2Conf.JS

 and I can see multiple errors in the console related to advanced-filters.js and select2:

![image](https://user-images.githubusercontent.com/2685808/150769754-790569f9-5def-45d5-85ad-89e3194ff336.png)

Do you plan to support new select2 version?

Reply:
I haven't been using the package myself recently; so I don't have any immediate plans for it. I should probably add a note, or apply to move it to jazzband...

Please create a separate ticket for the revent select2 version support

Reply:
Done: #164